install/0000_00_cluster-version-operator_01_admingate_configmap.yaml

```diff
@@ -1,9 +1,14 @@
 apiVersion: v1
 kind: ConfigMap
+data:
+  ack-4.11-kube-1.24-api-removals-in-4.12: |
```

Comment:
@LalatenduMohanty Shouldn't this be Kube 1.25 throughout?

Reply:
I agree, should be 1.25, e.g. see openshift/cluster-kube-apiserver-operator#1332 adjusting the 4.11 `APIRemovedInNextReleaseInUse` to complain about removed-in-1.25 APIs.

Reply:
Fixed in https://github.com/openshift/cluster-version-operator/commit/2d66c169658a72e1c54272646d352fc10f378b6f